server/recording_api.go

```diff
@@ -60,32 +68,26 @@ func (p *Plugin) recJobTimeoutChecker(callID, jobID string) {
 	}
 }
 
-func (p *Plugin) handleRecordingStartAction(state *channelState, callID, userID string) (*RecordingStateClient, httpResponse) {
-	var res httpResponse
-
+func (p *Plugin) startRecordingJob(state *channelState, callID, userID string) (rst *JobStateClient, rcode int, rerr error) {
```

Comment:
The reason to have named returns is so that we can intercept errors in the defer scope. But Go doesn't allow to have a single named return, it's either all of them or none, hence the change.

Reply:
Ahh, I missed that connection, thanks.